akka-parsing/src/main/scala/akka/macros/LogHelper.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.macros
```

Comment:
We don't currently support OSGI in akka-http. Shouldn't be a problem to move that internal class later on.